inspire/modules/workflows/workflows/process_record_arxiv.py

```diff
@@ -193,7 +193,7 @@ def get_sort_data(bwo, **kwargs):
             if decision == "CORE":
                 relevance_score += 10
             elif decision == "Rejected":
-                relevance_score -= 10
+                relevance_score = (max_score * -1) - 10
```

Comment:
This is a arbitrary value is used for sorting, so that when sorting by the score in Holding Pen it allows to sort by most relevant records. Say confident CORE predictions on top, and most confident Rejection at bottom.